modules/mod_ginger_foundation/templates/_script.tpl

```diff
@@ -30,7 +30,7 @@
 %}
 
 {% if m.modules.active.mod_geo %}
-    <script src="//maps.googleapis.com/maps/api/js?key={{ m.config.mod_geo.api_key.value|escape }}&amp;libraries=places&amp;language=nl&amp;v=3.31"></script>
+    <script src="//maps.googleapis.com/maps/api/js?key={{ m.config.mod_geo.api_key.value|escape }}&amp;libraries=places&amp;language=nl&amp;v=3.34"></script>
```

Comment:
Before, we pinned Google Maps on purpose: #390. Are you sure we can now upgrade without trouble?

Reply:
Good question! Ik will test this before removing WIP state.